lib/generators/rolemodel/slim/slim_generator.rb

```diff
@@ -0,0 +1,7 @@
+module Rolemodel
+  class SlimGenerator < Rails::Generators::Base
+    def add_slim
+      run 'bundle add slim'
```

Comment:
The `gem` action does not check to see if it's already in your gemfile, while `bundle add` does.  `bundle add` also runs `bundle install` and accepts an optional `--group` argument.  So, I believe this is the better mechanism for adding gem dependencies.

Reply:
👍